### ISSUE TITLE
Flash not working for single-stream capture

### ISSUE THREAD

**Issue by Kimblebee** [google/jetpack-camera-app#99]
Neither front nor rear flash is functional when the camera is set to single-stream capture mode

**Comment by temcguir** [comment]
Hi Kim, can you verify this issue still exists? I can't seem to reproduce it on device.

**Comment by Zeronfinity** [comment]
Probably related to [this CameraX issue regarding flash with OverlayEffect](https://issuetracker.google.com/issues/360641479) and [this issue regarding single-stream mode not really being single stream](https://github.com/google/jetpack-camera-app/issues/186).

The flash issue should happen only when ImageCapture is merged into StreamSharing. So the second issue is masking the first issue here.

**Comment by temcguir** [comment]
Do we need to file a bug for CameraX? Regardless of whether the ImageCapture is part of StreamSharing, I would still expect the flash to fire.

**Comment by Zeronfinity** [comment]
Right, this is ultimately a bug at CameraX side. There are already public bugs filed to CameraX for this and we are working on fixing the issue. It should be fixed soon.

https://issuetracker.google.com/issues/360641479
https://issuetracker.google.com/issues/297502589
